packages/flutter/lib/src/material/slider.dart

```diff
@@ -1220,3 +1317,62 @@ class _RenderSlider extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     }
   }
 }
+
+class _ValueIndicatorRenderObjectWidget extends LeafRenderObjectWidget {
+  const _ValueIndicatorRenderObjectWidget({
+    this.valueIndicatorAnimation,
```

Comment:
Same comment as in RangeSlider, this can probably be obtained from the state

Reply:
I think we could create the valueIndicatorAnimation from the other RenderBox since this animation is dependent on the Value Indicator Controller.  Would it be better to pass this variable through to the previous method or to just create a different Value Indicator Animation from the state value indicator controller

Reply:
I am not creating the value indicator animation from the other render box. That way it doesn't have to get passed through.